hermes-main-website/src/app/components/footer/footer.component.sass

```diff
@@ -0,0 +1,124 @@
+
+@import "src/variables.sass"
+
+
+
+.container
+  width: 100%
+  position: absolute
+  bottom: 0
+  color: $white-main
+  background-color:$black-main
+ 
+
+.footer-row
+  display: flex
+  align-items: flex-start
+  justify-content: center
+
+
+.text-align-center
+  margin-top: 20px 
+  margin-bottom:  20px 
+  text-align: center
+  color: $white-main
+  font-size: 30px
+  background-color: $black-main
+
+.footer--contact
+  color: $white-main
+  display: flex
+  flex-direction: column
+  margin: 0 20px
+  width: 33%
+  
+
+.footer--adress
```

Comment:
footer-address and footer-contact are pretty much the same, so create just one property "footer-column" and apply it to the div, keep align-items center for all here since it was my mistake to assume that they need left and right alignament